internal/process/process.go

```diff
@@ -48,13 +54,15 @@ func ActivationPID(cfg Configurable) int32 {
 			return -1
 		}
 
-		name, err := pproc.Ppid()
+		name, err := pproc.Name()
 		if err != nil {
 			logging.Error(procInfoErrMsgFmt, err)
 		} else {
 			logging.Debug("Parent process name: %s", name)
 		}
 
+		seen[pid] = true
```

Comment:
I think we should drop this one. We technically have not "seen" this one since we only look at the ppid. Sure the ppid that comes around might equal the pid, but in that case line 65 would already have handled that.